pkgs/top-level/all-packages.nix

```diff
@@ -4644,7 +4644,7 @@ with pkgs;
 
   clevercsv = with python3Packages; toPythonApplication clevercsv;
 
-  clevis = callPackage ../tools/security/clevis {
+  clevis = callPackage ../by-name/cl/clevis/package.nix {
     asciidoc = asciidoc-full;
   };
```

Comment:
We can probably inline this

Reply:
As @infinisil recommended, changes on the `override` interface should not happen at will.
At least I should test if asciidoc works fine before trying to inline it.

Reply:
Yeah I'd prefer to not inline these, since it would break `.override { asciidoc = ...; }`, which should be considered a relatively stable interface.